packages/typography/src/Typography.story.tsx

```diff
@@ -12,11 +12,14 @@ import {
   Label,
   Description,
   Link,
+  StaticWidthText,
```

Comment:
I think if you read this name for the first time you wouldn't understand what it does. I'm not sure if this is any better or not but maybe something like, `InteractiveTextWidth`?